src/modopt/opt/algorithms/admm.py

```diff
@@ -85,7 +87,7 @@ class ADMM(SetUp):
         Constraint vector
     optimizers: tuple
         2-tuple of callable, that are the optimizers for the u and v.
-        Each callable should access an init and obs argument and returns an estimate for:
```

Comment:
Not sure why you removed "an" here.

Reply:
To win a few chars and make the linter happy ^^